options/options.go

```diff
@@ -238,6 +260,21 @@ func (terragruntOptions *TerragruntOptions) AppendTerraformCliArgs(argsToAppend
 	terragruntOptions.TerraformCliArgs = append(terragruntOptions.TerraformCliArgs, argsToAppend...)
 }
 
+func (terragruntOptions *TerragruntOptions) GetIncludeConfig() *models.IncludeConfig {
+	var ret *models.IncludeConfig
+	ret = nil
+	if len(terragruntOptions.Context.Includes) > 0 {
+		ret = terragruntOptions.Context.Includes[0]
+	}
+	return ret
+}
+
+func (terragruntOptions *TerragruntOptions) AddIncludeConfig(includedConfig *models.IncludeConfig) {
+	if nil != includedConfig {
```

Comment:
`includedConfig != nil`

Reply:
Refer to my comment about Yoda conditions above

Reply:
Same as above